truss/templates/server/common/logging.py

```diff
@@ -5,11 +5,44 @@
 
 LEVEL: int = logging.INFO
 
+
+class CustomJsonFormatter(jsonlogger.JsonFormatter):
```

Comment:
Can you add some unit tests for this class that show it handles the various messages from the issue?